sql/test015.sql

```diff
@@ -0,0 +1,6 @@
+-- check that conjunctions correctly handle NULL values
+SELECT 14 AS test,
```

Comment:
I changed the `stitch`-script yesterday to generate the test numbering automatically. As such explicit numbering is no longer needed. 😁